### PR TITLE
User Guide Assets Section Part 4. 

### PR DESCRIPTION
**Pull request by micronAMZN**
Signed-off-by: Mike Cronin <mikecro@amazon.com>

## Change summary

This organizes all the Asset Processor specific topics into one section. Relies on images in PR# 1082.

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

File: content/docs/user-guide/assets/asset-processor/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{{< /note >}}
+
+## Asset Processor topics
+
+| Topic | Description |
+| - | - |
+| [Interface](interface) | Learn to navigate Asset Processor's interface, and how to read the status of process jobs. |
+| [Asset Processor Batch](asset-processor-batch) | With Asset Processor batch, you can batch process the source assets for a project in an automated build system. |
+| [Configuration](configuration) | Asset Processor can be configured by setting options in the `AssetProcessorPlatformConfig.setreg` configuration file. |
+| [Faster Scanning](faster-scannning) | Asset Processor's faster scanning mode uses source asset timestamps and quick source asset checks to determine if assets need to be processed. |
```
Comment (sptramer) [inline content/docs/user-guide/assets/asset-processor/_index.md:41]:
```suggestion
| [Faster Scanning](faster-scanning) | Asset Processor's faster scanning mode uses source asset timestamps and quick source asset checks to determine if assets need to be processed. |
```

Looks like there was a small typo in here.

**Review by sptramer — CHANGES_REQUESTED** [review]
One broken link, otherwise looks good!

**Review by sptramer — APPROVED** [review]

File: content/docs/user-guide/assets/asset-processor/interface.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+| - | - |
+| **Show in Asset Browser** | Highlights the asset in O3DE Editor’s Asset Browser, if O3DE Editor is open. |
+| **Open in Explorer** | Opens the asset in the system file browser. |
+| **Copy**  | Copies the asset name. |
+| **Open log file** | Opens the most recent log file for the asset, if one exists. |
+| **Open folder with log file** | Opens the directory containing the log file for the asset. |
+
+### Event Log Details
+
+The Event Log Details table displays asset processing information for the asset selected the Asset Status table. This table provides an activity log that shows how the asset was processed and any errors or warnings generated during processing.
```
Comment (carrcain) [inline content/docs/user-guide/assets/asset-processor/interface.md:106]:
```suggestion
The Event Log Details table displays asset processing information for the asset selected in the Asset Status table. This table provides an activity log that shows how the asset was processed and any errors or warnings generated during processing.

```

File: content/docs/user-guide/assets/asset-processor/interface.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+### Product Assets
+
+On the left, the Product Assets list displays all the product assets that match the current search filter. When the search is empty, all product assets are displayed.
+
+When an asset is selected in the Product Assets list, information about the asset is displayed in the panes on the right.
+
+| Pane | Description |
+| - | - |
+| **Asset Information** | Information for the product asset including the asset UUID, the last time the product was generated, the type of job that generated the asset, which platform the asset was produced for, and which source asset is the primary input for the product. |
+| **Outgoing Product Dependencies** | The list of product assets which depend on this product asset. |
```
Comment (carrcain) [inline content/docs/user-guide/assets/asset-processor/interface.md:180]:
```suggestion
| **Outgoing Product Dependencies** | The list of product assets that depend on this product asset. |

```

File: content/docs/user-guide/assets/asset-processor/interface.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+| **Outgoing Unmet Path Product Dependencies**<sup>1</sup> | The list of product assets which are hardcoded paths to be loaded by the O3DE runtime. |
```
Comment (carrcain) [inline content/docs/user-guide/assets/asset-processor/interface.md:181]:
```suggestion
| **Outgoing Unmet Path Product Dependencies**<sup>1</sup> | The list of product assets that are hardcoded paths to be loaded by the O3DE runtime. |

```

File: content/docs/user-guide/assets/asset-processor/interface.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+| **Incoming Product Dependencies** | The list of product assets which this product asset depends on. |
```
Comment (carrcain) [inline content/docs/user-guide/assets/asset-processor/interface.md:182]:
```suggestion
| **Incoming Product Dependencies** | The list of product assets that this product asset depends on. |

```

File: content/docs/user-guide/assets/asset-processor/interface.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+| **Missing Product Dependencies** | Detect missing product dependencies. |
+
+{{< note >}}
+<sup>1</sup>Because these products aren’t necessarily generated by Asset Processor, they’re placed into a separate category of dependencies.
+{{< /note >}}
+
+## Logs
+
+The Logs tab displays logs for Asset Processor itself, not for process jobs. If you are having issues with a specific process job, start debugging with the the logs for the job found in the Jobs tab. The type of errors displayed in the Logs tab are often related issues with managing the process jobs, maintaining the Asset Cache, and writing log information for process jobs.
```
Comment (carrcain) [inline content/docs/user-guide/assets/asset-processor/interface.md:191]:
```suggestion
The Logs tab displays logs for Asset Processor itself, not for process jobs. If you are having issues with a specific process job, start debugging with the logs for the job found in the Jobs tab. The type of errors displayed in the Logs tab are often related issues with managing the process jobs, maintaining the Asset Cache, and writing log information for process jobs. 

```

File: content/docs/user-guide/assets/asset-processor/debugging.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
+---
+linkTitle: Debugging
+title: Debugging Asset Processor Issues
+description: Methods to debug Asset Processor issues in Open 3D Engine (O3DE).
+weight: 500
+toc: true
+---
+
+If you are having issues with **Asset Processor** use the methods below to debug the issues.
```
Comment (carrcain) [inline content/docs/user-guide/assets/asset-processor/debugging.md:9]:
```suggestion
If you are having issues with **Asset Processor**, use the methods below to debug the issues.

```

File: content/docs/user-guide/assets/asset-processor/debugging.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+description: Methods to debug Asset Processor issues in Open 3D Engine (O3DE).
+weight: 500
+toc: true
+---
+
+If you are having issues with **Asset Processor** use the methods below to debug the issues.
+
+## View Asset Processor Logs 
+
+If Asset Processor isn't working as expected, use the information in the **Logs** tab to debug the issue. The Logs tab contains log information for Asset Processor and not for individual process jobs.
```
Comment (carrcain) [inline content/docs/user-guide/assets/asset-processor/debugging.md:13]: Point to where they can find debug info for individual jobs.

**Review by carrcain — CHANGES_REQUESTED** [review]
Editorial review.

**Comment by sptramer** [comment]
Mike addressed comments from AWS editorial, who do not have the time to re-review right now, so merging under the auspices of "Mike accepted the copyedit."

**Pull request merged**